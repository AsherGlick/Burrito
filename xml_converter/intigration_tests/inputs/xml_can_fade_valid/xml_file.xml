<OverlayData>
<MarkerCategory  Name="mycategory">
</MarkerCategory>

<POIs>
<POI CanFade="false" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" IconFile="texture.png" />
<POI CanFade="0" Type="mycategory" XPos="179.81" YPos="220.65" ZPos="225.83" MapID="50" IconFile="texture.png" />
<!-- TODO: This value should be elided in the output, but it is not currently
so we are ignoring the test instead of wriitng a bad test
<POI CanFade="true" Type="mycategory"/>
<POI CanFade="1" Type="mycategory"/> -->
</POIs>
</OverlayData>
