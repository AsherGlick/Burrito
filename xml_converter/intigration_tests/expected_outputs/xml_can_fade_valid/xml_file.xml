<OverlayData>
<MarkerCategory  Name="mycategory">
</MarkerCategory>

<POIs>
<POI  CanFade="false" Type="mycategory" IconFile="texture.png" MapID="50" XPos="169.809998" YPos="210.649994" ZPos="215.830002"/>
<POI  CanFade="false" Type="mycategory" IconFile="texture.png" MapID="50" XPos="179.809998" YPos="220.649994" ZPos="225.830002"/>
</POIs>
</OverlayData>
